src/main/java/bio/terra/cloudfiletodatastore/deltalayer/GcsFileFetcher.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.cloudfiletodatastore.deltalayer;
+
+import bio.terra.cloudfiletodatastore.ResourceFetcher;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import java.util.logging.Logger;
+
+public class GcsFileFetcher implements ResourceFetcher {
+
+  private final String bucket;
+  private final String name;
+
+  private static final Logger logger = Logger.getLogger(GcsFileFetcher.class.getName());
+
+  public GcsFileFetcher(String bucket, String name) {
+    this.bucket = bucket;
+    this.name = name;
+  }
+
+  @Override
+  public byte[] fetchResourceBytes() {
+    Storage service = StorageOptions.newBuilder().build().getService();
+    Blob blob = service.get(BlobId.of(bucket, name));
+    byte[] content = blob.getContent();
+    logger.info(String.format("Bytes length is %s", content.length));
+    return content;
+  }
```

Comment:
TODO either for this PR or for a future PR: add retries here to handle transient errors

Reply:
would you mind adding a `// TODO:` comment here about retries?